Detectors/TPC/workflow/src/ProcessingHelpers.cxx

```diff
@@ -61,3 +62,10 @@ uint64_t processing_helpers::getCreationTime(o2::framework::ProcessingContext& p
 {
   return DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
 }
+
```

Comment:
With my comments from above, moving code to `processing_helpers` please also add documentation, that the `InputSpec` for `ccdbParamSpec("CTP/Calib/OrbitReset")` must be defined.

Reply:
Yes you are right, I just fixed it and moved the function to the `processing_helpers`